jvb/src/main/kotlin/org/jitsi/videobridge/load_management/JvbLoadMeasurement.kt

```diff
@@ -16,6 +16,10 @@
 
 package org.jitsi.videobridge.load_management
 
+/**
+ * A measurement of load on the JVB.  Implementations should implement a description of the
+ * measurement in the [toString] method.
+ */
```

Comment:
You should probably follow your own advice in `MockLoadMeasurement`.

Reply:
Don't think it's really necessary there, but ok.